app/models/course/assessment/category.rb

```diff
@@ -8,10 +10,16 @@ class Course::Assessment::Category < ActiveRecord::Base
 
   accepts_nested_attributes_for :tabs
 
+  before_create :build_initial_tab
   before_destroy :validate_before_destroy
 
   default_scope { order(:weight) }
 
+  def self.after_course_create(course)
+    course.assessment_categories.
+      create(title: Course::Assessment::Category.human_attribute_name('title.default'), weight: 0)
```

Comment:
The key that this eventually points to has the value 'Assessments', which will be the name of the default category. Would it be better to use that directly, and skip the ActiveRecord keys?

The name of the first tab is also affected.